driver/bpf/fillers.h

```diff
@@ -4579,37 +4566,27 @@ FILLER(sys_ppoll_e, true)
 	unsigned long val;
 	int res;
 
+	/* Parameter 1: fds (type: PT_FDLIST) */
 	res = bpf_poll_parse_fds(data, true);
-	if (res != PPM_SUCCESS)
-		return res;
+	CHECK_RES(res);
 
-	/*
-	 * timeout
-	 */
+	/* Parameter 2: timeout (type: PT_RELTIME) */
 	val = bpf_syscall_get_argument(data, 2);
 
 	/* NULL timeout specified as 0xFFFFFF.... */
 	if (val == (unsigned long)NULL) {
 		res = bpf_val_to_ring_type(data, (u64)(-1), PT_RELTIME);
-		if (res != PPM_SUCCESS)
-			return res;
+		CHECK_RES(res);
```

Comment:
I'd move the `CHECK_RES` after the `if/else` check, so that we call it just once!